tools/msvs/msi/nodemsi.wixproj

```diff
@@ -72,7 +72,10 @@
     </HeatDirectory>
   </Target>
   <PropertyGroup>
-    <PostBuildEvent>move "!(TargetPath)" "$(TargetDir)\$(TargetFileName)"
-move "!(TargetPdbPath)" "$(TargetDir)\$(TargetPdbName)"</PostBuildEvent>
+    <PostBuildEvent>
+      "$(WixToolPath)\torch.exe" -t language "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)de-de\$(TargetFileName)" -out "$(TargetDir)transforms\de-de.mst"
+      cscript.exe WiSubStg.vbs "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)transforms\de-de.mst" 1031
```

Comment:
It embeds the extracted language information from de-de to the en-us installer as additional language (1031 = de-de, https://msdn.microsoft.com/en-us/goglobal/bb964664.aspx).